docs/unifiedpush/aerogear-push-android/google-setup.asciidoc

```diff
@@ -1,45 +1,40 @@
 [[google-setup]]
-== Google Related Setup
+== Obtaining Firebase Cloud Messaging Credentials
 
-Before the Android application is able to receive the notifications, you must authenticate yourself with Google as a developer. Luckily this is both easy and free to do, and doesn't require much set-up. Like iOS, you are only able to test out your push notifications in a real device.
+Before the Android application is able to receive the notifications, you must set up access to the Firebase Console. Luckily, this is both easy and free to do, and doesn't require much set-up. As with iOS, you are only able to test out your push notifications in a real device. This section shows you how to obtain the credentials necessary to set up Firebase Cloud Messaging for your app, including:
 
-=== Registering with Google
+* the Server key
+* the Sender ID
+* the `google-services.json` file that needs to be placed into the /app folder of your Android Studio project.
```

Comment:
I'd say _"that needs to be placed into the /app folder of your Android Studio project"_ is implementation-specific detail. For example, you may wish to use `google-services.json` with aerogear-cordova, which doesn't have an `/app` folder. If we want to put any description here, I'd say make it the description of the contents or the purpose of the file - credentials for access to Firebase and Google services.